Modules/_multiprocessing/multiprocessing.c

```diff
@@ -180,23 +161,23 @@ PyInit__multiprocessing(void)
         else
             py_sem_value_max = PyLong_FromLong(SEM_VALUE_MAX);
         if (py_sem_value_max == NULL)
-            return NULL;
+            return -1;
         PyDict_SetItemString(_PyMp_SemLockType.tp_dict, "SEM_VALUE_MAX",
```

Comment:
Please add error handling. By the value, it seems like Py_DECREF(py_sem_value_max) is needed.